x-pack/test/accessibility/apps/index_lifecycle_management.ts

```diff
@@ -55,6 +58,32 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await esClient.ilm.deleteLifecycle({ policy: TEST_POLICY_NAME });
     });
 
+    it('Create Policy Form', async () => {
+      await retry.waitFor('Index Lifecycle Policy create/edit view to be present', async () => {
+        return testSubjects.isDisplayed('createPolicyButton');
+      });
+
+      // Navigate to create policy page and take snapshot
+      await testSubjects.click('createPolicyButton');
+      await retry.waitFor('Index Lifecycle Policy create/edit view to be present', async () => {
+        return (await testSubjects.getVisibleText('policyTitle')) === 'Create policy';
+      });
+
+      // Fill out form after enabling all phases and take snapshot.
+      await indexLifecycleManagement.fillNewPolicyForm('testPolicy', true, true, false);
+      await a11y.testAppSnapshot();
+    });
+
+    it('Send Request Flyout on New Policy Page', async () => {
+      // Take snapshot of the show request panel
+      await testSubjects.click('requestButton');
+      await a11y.testAppSnapshot();
+
+      // Close panel and save policy
+      await testSubjects.click('euiFlyoutCloseButton');
+      await indexLifecycleManagement.saveNewPolicy();
```

Comment:
Do you think it might be worth waiting for the "list"/"table" view to load? That way we know the policy was successfully created, alternatively we can also check for the toast?